servicetalk-concurrent-api/src/main/java/io/servicetalk/concurrent/api/Completable.java

```diff
@@ -1895,9 +1899,10 @@ public static Completable mergeAllDelayError(int maxConcurrency, Completable...
      *      }
      * }</pre>
      *
-     * @param completables {@link Completable}s the result of which are to be ambiguated.
+     * @param completables {@link Completable}s to subscribe to and race to propagate to the return value.
```

Comment:
Perhaps "result" rather than "return value".

Reply:
We use "return value" elsewhere in these docs, so I'll use that for consistency.